docs/docusaurus/docs/cloud/setup/setup_cloud.md

```diff
@@ -0,0 +1,183 @@
+---
+sidebar_label: Try GX Cloud
+title: Try GX Cloud
+id: setup_cloud
+description: An end-to-end reference to help new users set up GX Cloud.
+---
+
+import TechnicalTag from '@site/docs/term_tags/_tag.mdx';
+import SetupAndInstallForSqlData from '/docs/components/setup/link_lists/_setup_and_install_for_sql_data.md'
+import SetupAndInstallForFilesystemData from '/docs/components/setup/link_lists/_setup_and_install_for_filesystem_data.md'
+import SetupAndInstallForHostedData from '/docs/components/setup/link_lists/_setup_and_install_for_hosted_data.md'
+import SetupAndInstallForCloudData from '/docs/components/setup/link_lists/_setup_and_install_for_cloud_data.md'
+
+If you're new to GX Cloud, the information provided here is intended to demonstrate GX Cloud features and functionality. You'll connect to a Data Source, build an Expectation using sample Batch data, validate data with the Expectation, and review Validation Results.
+
+After you've tested GX Cloud features and functionality, you can connect to your Data Source and create Expectations that are specific to your business needs.
+
+Although you can use any Python Interpreter or script file to run Python code, Great Expectations (GX) recommends using Jupyter Notebook. Jupyter Notebook is included with OSS GX and is the best option for composing script files and running code.
+
+The example code is available in the [onboarding script repository](https://github.com/great-expectations/great_expectations/blob/develop/assets/scripts/gx_cloud/experimental/onboarding_script.py).
+
+## Prerequisites
+
+- An active [GX Cloud Beta Account](https://greatexpectations.io/cloud).
+
+## Prepare your environment
+
+1. Download and install the latest Python version. See [Active Python Releases](https://www.python.org/downloads/).
+
+2. Download and install pip. See the [pip documentation](https://pip.pypa.io/en/stable/cli/pip/).
+
+3. Run the following command in an empty base directory inside a Python virtual environment:
+
+    ```bash title="Terminal input"
+    pip install great_expectations
+    ```
+
+    It can take several minutes for the installation to complete.
+
+    If you've previously installed GX, run the following command to upgrade to the latest version:
+
+    ```bash title="Terminal input"
+    pip install great_expectations --upgrade
+    ```
+4. In Jupyter Notebook, run the following Python code to import the modules you'll use to test functionality:
```

Comment:
As a developer, I find it counterintuitive to see the notebook imports separated from the remainder of the code I'll be using in the notebook. My recommendation is that environment steps 1 - 3 (download python, pip, install great_expectations) belong in the "Prepare your environment" section, but the imports belong in a section/step right before "Create a Data Context", such that the overall flow for a user is:
* Set up the Python environment (python, pip)
* Go to the Cloud, get credentials
* Spin up Jupyter Notebook
* Write and run the walkthrough code in the notebook

Reply:
@rachhouse good feedback. I initially had the module installation as a standalone section. In an attempt to reduce the number of sections, I added the module installation to the _Prepare_ procedure. I agree with your feedback and I'll return the module installation to a standalone section.